test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/test_tfs.py

```diff
@@ -113,6 +113,10 @@ def test_tfs_neuron_model(boto_session, sagemaker_client,
 def test_batch_transform(region, boto_session, sagemaker_client,
                          model_name, tfs_model, image_uri,
                          instance_type):
+                         
+    if "c6" in instance_type:
```

Comment:
Is this skip condition specific to c6g instances or specific to graviton images? If the former, please make this more selective - you can use regular expressions and make sure to look for "c6g" as there are different types of c6 instances.

If the latter, you should skip the test if the image URI has graviton, so that the same logic won't need to be added if instance types change in the future.

Reply:
will use image_uri.